library/fluid/fluid-flow/build.gradle

```
@@ -0,0 +1,12 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	moduleName = "fluid-flow"
```

```suggestion
	moduleName = "fluid_flow"
```

All modules name are to be snake_cased.